releases/release-4.0.8.md

```diff
@@ -0,0 +1,159 @@
+    - Add system table `table_storage_stats` [#20431](https://github.com/pingcap/tidb/pull/20431)
+    - Add rpc runtime stats information for insert/update/replace statement. [#20430](https://github.com/pingcap/tidb/pull/20430)
+    - Add executor information in `explain for connection` result. [#20384](https://github.com/pingcap/tidb/pull/20384)
+    - Fix a bug that the coercibilities of enum and set are wrong. [#20364](https://github.com/pingcap/tidb/pull/20364)
+    - Config: check if valid storage by simple indexing [#20342](https://github.com/pingcap/tidb/pull/20342)
+    - Duplicate order by conditions are eliminated [#20333](https://github.com/pingcap/tidb/pull/20333)
+    - The TiDB error log now reports client connect/disconnect activity only under debug level verbosity. [#20321](https://github.com/pingcap/tidb/pull/20321)
+    - Add metrics for coprocessor cache [#20293](https://github.com/pingcap/tidb/pull/20293)
+    - Fix ambiguous year conversion [#20292](https://github.com/pingcap/tidb/pull/20292)
+    - Fixed an issue where the KV duration panel contains `store0` [#20260](https://github.com/pingcap/tidb/pull/20260)
+    - Fix FLOAT data type: out of range data should not be inserted [#20252](https://github.com/pingcap/tidb/pull/20252)
+    - Fix a bug that the generated column doesn't handle bad null value [#20216](https://github.com/pingcap/tidb/pull/20216)
+    - Add pessimistic lock keys runtime information [#20199](https://github.com/pingcap/tidb/pull/20199)
+    - Add two extra time-consuming sections in runtime information and trace span [#20187](https://github.com/pingcap/tidb/pull/20187)
+    - Add transaction commit runtime information in slow log [#20185](https://github.com/pingcap/tidb/pull/20185)
+    - Fix inaccurate error info for year column out of range. [#20170](https://github.com/pingcap/tidb/pull/20170)
+    - Fix unexpected 'invalid auto-id' error in pessimistic txn retry. [#20134](https://github.com/pingcap/tidb/pull/20134)
+    - Fix an issue that alter enum/set type does not check constraint [#20046](https://github.com/pingcap/tidb/pull/20046)
+    - Fix cop task runtime information is wrong in the concurrent executor. [#19947](https://github.com/pingcap/tidb/pull/19947)
+    - Fix cannot select session scope explicitly with unchangeable variables [#19944](https://github.com/pingcap/tidb/pull/19944)
+
++ TiKV
+
+    - Error_code: re-format metafile [#8877](https://github.com/tikv/tikv/pull/8877)
+    - Move counter metric processing into the scheduler worker. [#8872](https://github.com/tikv/tikv/pull/8872)
+    - Record the noop entry message temporarily like pending_votes in raftstore [#8864](https://github.com/tikv/tikv/pull/8864)
+    - Enable dynamically change config pessimistic-txn.pipelined [#8853](https://github.com/tikv/tikv/pull/8853)
+    - Enabling profiling features by default [#8801](https://github.com/tikv/tikv/pull/8801)
```

Comment:
Move them to `Improvements`

Reply:
And also remove the `Move counter metric processing into the scheduler worker. [#8872](https://github.com/tikv/tikv/pull/8872)`

Reply:
Move `Record the noop entry message...` to bug fix pard and change it to `Fix the bug that a learner can't find a leader after restarting or splitting`